common/chunkStatusLogger.go

```diff
@@ -315,6 +316,18 @@ func (csl *chunkStatusLogger) FlushLog() {
 	}
 }
 
+// CloseLogger close the chunklogger thread.
+func (csl *chunkStatusLogger) CloseLogger() {
+	// Once logger is closed, we log no more chunks.
+	csl.outputEnabled = false
```

Comment:
This is not thread safe. Some logs will be lost. Is it acceptable?

Reply:
This is exposed in ChunkStatusLoggerCloser() interface which is known to JobManager. It's being called from JobManager only exported function DeferredCleanupJobMgr() func. Before calling DeferredCleanupJobMgr we remove jobMgr entry from jobsAdmin.